spdk.spec

```diff
@@ -152,6 +148,9 @@ make -C doc
 
 
 %install
+# Workaround for https://github.com/spdk/spdk/issue/2531
+sed -i -e 's/\(-L\$1\/\)lib/\1%{_lib}/' scripts/pc.sh
+
```

Comment:
My apologies.  This should go in the `%prep` section, not `%install` section.

Reply:
https://build.hpdd.intel.com/blue/organizations/jenkins/daos-stack%2Fspdk/detail/PR-47/24/pipeline/186 past build after reverting the prefix change and adding the workaround: commit https://build.hpdd.intel.com/blue/organizations/jenkins/daos-stack%2Fspdk/detail/PR-47/24/pipeline/186